src/Store.js

```diff
@@ -500,4 +501,42 @@ export default class Store {
         }
         return Promise.all(promises);
     }
+
+    save(options = {}) {
+        return this._saveAll(options);
+    }
+
+    @action
+    _saveAll(options = {}) {
+        this.clearValidationErrors();
+        return this.wrapPendingRequestCount(
+            this.__getApi()
+                .saveAllModels({
+                    url: result(this, 'url'),
+                    model: this,
+                    data: this.toBackendAll({
+                        data: options.data,
+                        mapData: options.mapData,
+                        nestedRelations: relationsToNestedKeys(options.relations || []),
+                        onlyChanges: options.onlyChanges,
+                    }),
+                    requestOptions: omit(options, 'relations', 'data', 'mapData'),
+                })
+                .then(action(res => {
+                    const promises = [];
+                    for (const model of this.models) {
+                        promises.push(model._afterSaveAll(res, options));
+                    }
+                    return Promise.all(promises);
```

Comment:
It should also call this.clearSetChanges after successful save, otherwise this.hasUserChanges will always stay true even after saving the store. I encountered this issue in a project that already uses changes from this PR